scripts/Steganography/main.py

```diff
@@ -0,0 +1,32 @@
+from LSBSteg import *
+from voice import *
+import voice
+import os
+
+
+print("--------------------------------------------------------------\n")
+print(" ENCODING THE EMAIL AND PASSWORD TO THE IMAGE\n")
+steg = LSBSteg(cv2.imread("my.jpg"))
+txt = str(input(" Enter the text you want to encrypt : "))
+img_encoded = steg.encode_text(txt)
+
+cv2.imwrite("my_new_image.png", img_encoded)
+print(
+    "\n The size of the image before encryption : {} bytes".format(
+        os.stat("my.jpg").st_size
+    )
+)
+
+
+print("--------------------------------------------------------------\n")
+print(" DECODING THE EMAIL AND PASSWORD FROM THE IMAGE\n")

```

Comment:
Likewise decoding message from image.

Reply:
Also the decoding should be made such that it is triggered only when the user wishes to decode a particular image file. And in case there is no message hidden in the image, the program should return an exception or message notifying the user about the same.